package.json

```diff
@@ -87,12 +88,15 @@
   },
   "pre-commit": "lint:staged",
   "dependencies": {
+    "@babel/preset-typescript": "^7.18.6",
```

Comment:
Why do we need this ?

Reply:
@storybook/test-runner need this .

Reply:
okay then should be moved to dev dependency